fineract-provider/src/main/resources/db/changelog/tenant/parts/0025_add_delinquency_buckets.xml

```diff
@@ -0,0 +1,205 @@
+        </createTable>
+    </changeSet>
+    <changeSet author="fineract" id="4">
+        <addForeignKeyConstraint baseColumnNames="delinquency_range_id" baseTableName="m_delinquency_bucket_mappings"
+                                 constraintName="FK_m_delinquency_range_mapping" deferrable="false"
+                                 initiallyDeferred="false" onDelete="RESTRICT" onUpdate="RESTRICT"
+                                 referencedColumnNames="id" referencedTableName="m_delinquency_range" validate="true"/>
+        <addForeignKeyConstraint baseColumnNames="delinquency_bucket_id" baseTableName="m_delinquency_bucket_mappings"
+                                 constraintName="FK_m_delinquency_bucket_mapping" deferrable="false"
+                                 initiallyDeferred="false" onDelete="RESTRICT" onUpdate="RESTRICT"
+                                 referencedColumnNames="id" referencedTableName="m_delinquency_bucket" validate="true"/>
+    </changeSet>
+    <changeSet author="fineract" id="5">
+        <addColumn tableName="m_product_loan">
+            <column name="delinquency_bucket_id" type="BIGINT" defaultValueComputed="NULL"/>
+        </addColumn>
+        <addForeignKeyConstraint baseColumnNames="delinquency_bucket_id" baseTableName="m_product_loan"
+                                 constraintName="FK_m_product_loan_m_delinquency_bucket" deferrable="false"
+                                 initiallyDeferred="false" onDelete="RESTRICT" onUpdate="RESTRICT"
+                                 referencedColumnNames="id" referencedTableName="m_delinquency_bucket" validate="true"/>
+    </changeSet>
+    <changeSet author="fineract" id="6">
+        <addColumn tableName="m_loan">
+            <column name="delinquency_range_id" type="BIGINT" defaultValueComputed="NULL"/>
+        </addColumn>
+        <addForeignKeyConstraint baseColumnNames="delinquency_range_id" baseTableName="m_loan"
+                                 constraintName="FK_m_delinquency_range_loan" deferrable="false"
+                                 initiallyDeferred="false" onDelete="RESTRICT" onUpdate="RESTRICT"
+                                 referencedColumnNames="id" referencedTableName="m_delinquency_range" validate="true"/>
+    </changeSet>
+    <changeSet author="fineract" id="7">
+        <createTable tableName="m_loan_delinquency_tags">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="delinquency_range_id" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="loan_id" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_by" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_on_utc" type="DATETIME">
```

Comment:
@josehernandezfintecheandomx For postgres the columnType should be "TIMESTAMP WITH TIME ZONE"

Reply:
Done!